google-cloud-pub-sub/src/main/scala/akka/stream/alpakka/googlecloud/pubsub/model.scala

```diff
@@ -99,9 +100,18 @@ final case class PubSubMessage(data: String,
 
 object PubSubMessage {
 
+  def apply(data: String): PubSubMessage = PubSubMessage(data, "")
+
+  def apply(data: String, attributes: immutable.Map[String, String]): PubSubMessage =
+    PubSubMessage(data, "", Some(attributes), None)
+
   /**
    * Java API: create [[PubSubMessage]]
    */
+  def create(data: String) =
+    PubSubMessage(data)
+
+  @deprecated("Setting messageId when creating message for publishing is futile.", "1.0.3")
```

Comment:
The version should be 1.1.0, as this is the version we are aiming to release in the middle of this week.